packages/suite/src/support/messages.ts

```diff
@@ -1416,7 +1416,7 @@ export default defineMessages({
         id: 'TR_COINMARKET_DCA_DOWNLOAD',
     },
     TR_COINMARKET_BUY_AND_SELL: {
-        defaultMessage: 'Buy & sell',
+        defaultMessage: 'Buy & Sell',
```

Comment:
I will nicely ask @MiroslavProchazka 🎅

Reply:
Done, it will be effective once the next Crowdin sync is completed.

<img width="710" alt="image" src="https://github.com/user-attachments/assets/bf1757da-984d-40ee-8c88-fa00dcbc03e7">